ngraph_bridge/ngraph_encapsulate_op.cc

```diff
@@ -148,9 +148,17 @@ void NGraphEncapsulateOp::CreateParallelExecutor(OpKernelConstruction* ctx,
   int graph_id{-1};
   OP_REQUIRES_OK(ctx, ctx->GetAttr("ngraph_graph_id", &graph_id));
 
+  int my_function_cache_depth_in_items = 16;
```

Comment:
can change 16 to a const, you can use something like this, which is property of cache
https://github.com/tensorflow/ngraph-bridge/blob/master/ngraph_bridge/ngraph_prefetch_shared_data.h#L60

Reply:
`my`->`m`